packages/toolkit/package.json

```diff
@@ -19,9 +19,6 @@
   },
   "bundleDependencies": [],
   "dependencies": {
-    "@10up/babel-preset-default": "^2.0.1-next.1",
-    "@10up/eslint-config": "^2.4.4-next.1",
```

Comment:
Thoughts on excluding the peer dependencies of eslint-config from this list as well?

I'd just leave `prettier` looked to a specific version to avoid projects inadvertently updating prettier and thus breaking eslint.

Reply:
If we trust semver (I think we should) we shouldn't ever run into that situation since breaking changes should be a part of a future major release. Given it's already a `peerDependency` it can be successfully locked at the project level at any version that's at least `2.4.1` without raising any warnings. 

Let me know if you don't agree with this and we can discuss this further since there could be something I'm missing!

Reply:
I took the suggestion to move things to peerDeps with Eslint! Thanks for catching that!